.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '5 5 * * 1'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
```

Comment:
why would you need this ? it should be executable in your git project already ?

Reply:
@tgeens  this is default when you start a Gradle Github action.